platform/mellanox/mlnx-platform-api/sonic_platform/psu.py

```diff
@@ -16,6 +16,9 @@
 except ImportError as e:
     raise ImportError (str(e) + "- required module not found")
 
+LED_ON = 1
```

Comment:
maybe define it as a string? then can avoid to convert it to string when reference it

Reply:
fixed